### ISSUE TITLE
MD013: opt-in sentence endings at terminal link text or image alt punctuation

### ISSUE THREAD

**Issue by ziyuEngineer** [rvben/rumdl#884]
## Feature request

Allow an opt-in sentence-boundary policy that recognizes terminal punctuation
at the **end** of link text or image alt text as ending the surrounding
sentence, while keeping the complete link/image source span atomic.

I understand the current documented policy excludes punctuation inside
links/images. This requests a narrowly scoped extension, not a report that
the existing implementation violates its documentation.

## Current behavior

Official Windows x86_64 binary, **v0.2.73**:

```toml
[global]
enable = ["MD013"]

[MD013]
line-length = 0
reflow = true
reflow-mode = "sentence-per-line"
```

These paragraphs remain unchanged after
`rumdl fmt --config .rumdl.toml --no-cache test.md`:

```markdown
Read [Done.](https://example.test) Next sentence.

See ![Done.](image.png) Next sentence.

查看 ![完成。](image.png) 下一句开始。
```

Under the proposed policy, the desired outputs would be:

```markdown
Read [Done.](https://example.test)
Next sentence.

See ![Done.](image.png)
Next sentence.

查看 ![完成。](image.png)
下一句开始。
```

The break is **after the complete construct**, never within its label, alt,
destination or title.

## Negative controls

This currently works and must continue working:

```markdown
See ![Done](image.png). Next sentence.
```

becomes:

```markdown
See ![Done](image.png).
Next sentence.
```

This must not acquire a sentence split inside the image or after `Done.`:

```markdown
See ![Done. More words](image.png) before continuing. Next sentence.
```

The current output correctly splits only before `Next sentence.`.

## Scope and safety

- Punctuation in a destination, title, code span or HTML attribute remains
  opaque and must not contribute sentence boundaries.
- Reuse sentence rules for abbreviations, decimals and ellipses.
- Handle trailing inline delimiters without splitting constructs.
- Only consider an actual parser-recognized terminal label/alt boundary.
- A label or alt ending in a period is not always a sentence, so an explicit
  policy and false-positive tests are preferable to changing the default.
- Preserve rendered link text, image alt, URLs, titles and nesting.

Related #853 concerns joining link-only continuation lines; that behavior
works in my v0.2.73 test. This is a different boundary: the punctuation sits
at the end of a complete link/image. It also supports #877 without treating
the entire remaining paragraph as one indivisible sentence.
